tests/pdfbox/runtest

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+set -ex
+
+VERSION=2.0.27
+APP=pdfbox-${VERSION}
+MAVEN_DOCKER=maven:3.9.0-eclipse-temurin-8
+TEST_DIR=`pwd`
+
+if [ ! -d ${APP} ]; then
+    curl -o ${APP}-src.zip  https://dlcdn.apache.org/pdfbox/${VERSION}/${APP}-src.zip
+    (unzip -o ${APP}-src.zip && cd ${APP} && git apply ../TestCreateSignature.patch)
+fi
+
+# create the chiselled JRE docker with maven
+docker build -t chiselled-maven \
+    --build-arg MAVEN_DOCKER=${MAVEN_DOCKER} \
+    --build-arg BASE_IMAGE=${BASE_IMAGE:-ubuntu/chiselled-jre:8_edge} \
+    .
+
+# build and tests with chiselled jre
+docker run --rm  \
+    --tmpfs /tmp:exec \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
```

Comment:
why do you want to run it as the host USER? Typically, using `-u` with IDs will never fail (unless for access permissions). So maybe a better sanity test could be to ensure the `app` user actually exists inside the image, by running it with `-u app`?

Reply:
The build writes to the local repository directory `.m2`. There are some options, but i think best if I tweak test.yml to use current user's id when building docker and then we can both test presence of the user and avoid permissions issues.